### PR TITLE
Allow community admin to enable stake in existing communities

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6684

## Description of Changes
- Allows community admins to enable stakes in existing communities (that are based on a stake enabled chain)

https://github.com/hicommonwealth/commonwealth/assets/51641047/b3fa7791-a009-4edc-92ba-122e56498d2a

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_EXISTING_COMMUNITY_STAKE_INTEGRATION_ENABLED`
- Create a new community, with a base chain that supports community stakes, but don't enable stake in the community creation process.
- Now visit `/:scope/manage/integrations` screen of the community and verify you see a "Stake" enablement section.
- Click on the `Enable Stake` button and verify you get redirected to `/:scope/manage/integrations/stake` screen 
- There click on the `No` button and verify you get redirected back
- Enable community stake following the steps (namespace reservation, transaction signing, launching stake) and verify you get redirected back to `/:scope/manage/integrations`
- Verify the `Stake` section on the `/:scope/manage/integrations` changes to an enabled stake status.
- Clicking on the "View Stake Information" button should redirect you to `/:scope/manage/integrations` where you can now see the contract info.

## Deployment Plan
Enable `FLAG_EXISTING_COMMUNITY_STAKE_INTEGRATION_ENABLED`

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx [hunk 1]
```diff
@@ -21,19 +29,41 @@ const StakeIntegration = () => {
     return <PageNotFound />;
   }
 
+  const community = app.chain.meta;
+  const communityChainId = `${
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx:33]: do we need template string here?

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx [hunk 2]
```diff
@@ -21,19 +29,41 @@ const StakeIntegration = () => {
...
+    community.ChainNode?.ethChainId || community.ChainNode?.cosmosChainId
+  }`;
+  const selectedAddress = app.user.addresses.find(
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx:36]: I think you can use this `useUserActiveAccount`

**Review by burtonator — APPROVED** [review]

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx:33]: it needs to be a string, so i added it

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx (hunk 2 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx:36]: we actually need an instance of `AddressInfo` here, the `userUserActiveAccount` returns an `Account` instance

**Pull request merged**